### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by Mister-Hope**
Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -469,6 +465,31 @@ export default {
 
 </details>
 
+### `emit` Options
+
+<details>
+<summary>
+❌ Typescript Defining validators in <code>emit</code> options for emit event in <code>defineComponent</code> options will not work. This is just an alignment to Vue3.
```
Comment (antfu) [inline README.md:472]:
```suggestion
❌ <code>emit</code> option is provided in type-level only, in order to align with Vue 3's type interface. Does NOT have actual effects on the code.
```

File: README.zh-CN.md [hunk 2]
```diff
@@ -423,6 +444,31 @@ export default {
 
 </details>
 
+### `emit` 选项
+
+<details>
+<summary>
+❌ TS: 在 <code>defineComponent</code> 选项中添加 <code>emit</code> 事件校验将不会有任何效果, 这仅为针对 Vue3 的选项对齐。
```
Comment (antfu) [inline README.zh-CN.md:451]: Can you help update this as well? Thanks

Comment (Mister-Hope) [reply README.zh-CN.md:451]: Sure :smile:

Comment (Mister-Hope) [reply README.zh-CN.md]: Done!

**Review by antfu — APPROVED** [review]

**Comment by antfu** [comment]
Thanks

**Comment by Mister-Hope** [comment]
My pleasure, could you please review my newest PR? It's affecting my current job.

**Pull request merged**